test/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -844,7 +844,7 @@ public void TestComputations()
             Assert.Equal(100.0, df.Columns["Double"].Max());
             Assert.Equal(-10.0f, df.Columns["Float"].Min());
             Assert.Equal((uint)0, df.Columns["Uint"].Product());
-            Assert.Equal((ushort)140, df.Columns["Ushort"].Sum());
```

Comment:
{Ushort: 0 1 3 6 10 null 16 23 31 40 }

0+1+3+6+10+16+23+31+40 = 130

Reply:
@JakeRadMSFT is this how Pandas handles math with null data? We should match whatever they do.